Docker/upload_dockerhub/publish.sh

```diff
@@ -29,22 +36,25 @@ export DOCKER_HOST=unix:///run/user/1000/docker.sock
 sleep 5
 
 echo 'Start tagging & publishing ...'
+
 # Login
-echo $TEST_DOCKER_HUB_ACCESS_TOKEN | docker login --username $DOCKER_HUB_ID --password-stdin
+echo "Signing in as $DOCKER_HUB_ID"
+echo $DOCKER_HUB_ACCESS_TOKEN | docker login --username $DOCKER_HUB_ID --password-stdin
+
 
 # load images
 docker load -i "$WIKIBASE_DOCKER_PATH"
 docker load -i "$QUERYSERVICE_BACKEND_DOCKER_PATH"
 docker load -i "$QUERYSERVICE_FRONTEND_DOCKER_PATH"
 
 # Tag Queryservice UI with version
-tag_and_push $QUERYSERVICE_UI_IMAGE_NAME $WIKIBASE_BRANCH_NAME
+tag_and_push $QUERYSERVICE_UI_IMAGE_NAME $RELEASE_MAJOR_VERSION
 
 # Tag Queryservice with version
-tag_and_push $QUERYSERVICE_IMAGE_NAME $WIKIBASE_BRANCH_NAME
+#tag_and_push $QUERYSERVICE_IMAGE_NAME $RELEASE_MAJOR_VERSION
```

Comment:
Was this (and below) left commented out on purpose?

Reply:
yes, i just want to try to delete one thing rather than three :)